src/common/MarkLine/MarkLine.tsx

```diff
@@ -11,7 +11,7 @@ export interface MarkLineProps {
 export const MarkLine: FC<Partial<MarkLineProps>> = ({
   pointX,
   height,
-  strokeWidth = 1,
+  strokeWidth = 0,
```

Comment:
Default props should be in the `MarkLine.defaultProps = {}` after the component definition. Otherwise they won't get bound correctly.